### ISSUE TITLE
Virtual router timezone always set to UTC instead of local timezone

### ISSUE THREAD

**Issue by onitake** [apache/cloudstack#3708]
##### ISSUE TYPE

 * Improvement Request

##### COMPONENT NAME

~~~
VR
~~~

##### CLOUDSTACK VERSION

~~~
4.11.3
~~~

##### CONFIGURATION

Local timezone: CET/CEST

##### OS / ENVIRONMENT

N/A

##### SUMMARY

The Virtual Router has its timezone set to UTC implicitly. While this provides a sane default, it's not necessarily the best choice in every environment.

In particular, when debugging issues, the UTC timestamps can make it difficult to track events across multiple systems.

##### STEPS TO REPRODUCE

The management server would propagate its local time zone into the host config of each VR.

Alternatively, a new global time zone setting would be introduced, that is used to set the VR time zone.

##### EXPECTED RESULTS

Each CloudStack component has a consistent local time zone set.

##### ACTUAL RESULTS

The virtual router always logs events with UTC timestamps.

**Comment by DaanHoogland** [comment]
if you expect the MS local time-zone to be on every VR this is a straightforward feature to implement. I don't think this is very useful for most clouds as they span time-zones (even corporate ones do at times) this would make it a per-zone setting.

**Comment by DaanHoogland** [comment]
@onitake are you planning any implementation (for next release)?

**Comment by DaanHoogland** [comment]
No-one seems to be interested in implementing this. Closing untill new interest is spawned.
